Traceback (most recent call last):
  File "/Users/mhjensen/opt/anaconda3/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/mhjensen/opt/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 1087, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/mhjensen/opt/anaconda3/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/mhjensen/opt/anaconda3/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/Users/mhjensen/opt/anaconda3/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/Users/mhjensen/opt/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 540, in async_execute
    await self.async_execute_cell(
  File "/Users/mhjensen/opt/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 832, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/mhjensen/opt/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 740, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
    ax = -D*vx[i]*abs(v[i])/m
    ay = -g - D*vy[i]*abs(v[i])/m
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
[0;32m<ipython-input-14-847092aec374>[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0max[0m [0;34m=[0m [0;34m-[0m[0mD[0m[0;34m*[0m[0mvx[0m[0;34m[[0m[0mi[0m[0;34m][0m[0;34m*[0m[0mabs[0m[0;34m([0m[0mv[0m[0;34m[[0m[0mi[0m[0;34m][0m[0;34m)[0m[0;34m/[0m[0mm[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      2[0m [0may[0m [0;34m=[0m [0;34m-[0m[0mg[0m [0;34m-[0m [0mD[0m[0;34m*[0m[0mvy[0m[0;34m[[0m[0mi[0m[0;34m][0m[0;34m*[0m[0mabs[0m[0;34m([0m[0mv[0m[0;34m[[0m[0mi[0m[0;34m][0m[0;34m)[0m[0;34m/[0m[0mm[0m[0;34m[0m[0;34m[0m[0m

[0;31mNameError[0m: name 'vx' is not defined
NameError: name 'vx' is not defined

